Web/Cookie.hs

```diff
@@ -12,6 +12,8 @@ module Web.Cookie
     , setCookieDomain
     , setCookieHttpOnly
     , setCookieSecure
+    , setCookieSameSite
+    , SameSiteOption(..)
```

Comment:
That sounds good to me. Added in https://github.com/snoyberg/cookie/pull/13/commits/4ca56737be97ba8e4bf6a9214d51edfbd9096ab0